gradle/verification-metadata.xml

```diff
@@ -491,6 +491,14 @@
             <sha256 value="d1f155830a5b06af7e1da0aaef5226b811087fb58c48bf8c14ac7fdaa92c21a0" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="androidx.databinding" name="databinding-common" version="8.0.2">
```

Comment:
@OscarSpruit looking at the changes, I only see added lines here, do you know if the command we're running only adds the missing ones or should also remove the unnecessary ones if needed?